cypress.json

```diff
@@ -0,0 +1,9 @@
+{
+  "projectId": "xysa6x",
```

Comment:
You need your own id https://dashboard.cypress.io/organizations/07c47405-2e3c-4482-bc47-727f1737f458/integrations/github
But it's a bit bugged :thinking: 
I cannot enable this repo right now.

Reply:
Thanks.

Reply:
I also added the CYPRESS_RECORD_KEY to the secrets :)